reco_utils/tuning/nni/svd_training.py

```diff
@@ -10,7 +10,7 @@
 import nni
 import surprise
 import sys
-sys.path.append("../../")
+sys.path.append("../../../")
```

Comment:
is this needed?

Reply:
yeah it is needed, it won't work otherwise